release/docker/Dockerfile

```diff
@@ -9,26 +9,26 @@ COPY $WHEEL_MITMPROXY /home/mitmproxy/
 
 # Add our user first to make sure the ID get assigned consistently,
 # regardless of whatever dependencies get added.
-RUN addgroup -S mitmproxy && adduser -S -G mitmproxy mitmproxy \
-    && apk add --no-cache \
-        su-exec \
+RUN groupadd mitmproxy && useradd mitmproxy -g mitmproxy\
+    && apt-get update \
+    && apt-get install -y\
         git \
         g++ \
-        libffi \
+        # libffi \
         libffi-dev \
-        libstdc++ \
+        # libstdc++\
         openssl \
-        openssl-dev \
+        libssl-dev \
         python3 \
```

Comment:
Since the base image is already based on python3, there is no need to (re-)install the package here.